dist/presence.js

```diff
@@ -0,0 +1,155 @@
+const presence = new Presence({
+    clientId: "829371771702345759"
+});
+let browsingStamp = Math.floor(Date.now() / 1000), title;
```

Comment:
Merge with previous const unless reassigned (which you don't)

Reply:
getting compilation errors, will get back to this one.